scribejava-apis/src/main/java/com/github/scribejava/apis/service/GeniusOAuthServiceImpl.java

```diff
@@ -0,0 +1,39 @@
+package com.github.scribejava.apis.service;
+
+import com.github.scribejava.core.builder.api.DefaultApi20;
+import com.github.scribejava.core.model.AbstractRequest;
+import com.github.scribejava.core.model.OAuth2AccessToken;
+import com.github.scribejava.core.model.OAuthConfig;
+import com.github.scribejava.core.model.OAuthConstants;
+import com.github.scribejava.core.oauth.OAuth20Service;
+
+public class GeniusOAuthServiceImpl extends OAuth20Service {
+	private static final String USER_AGENT = "user-agent";
+	private static final String AGENT_NAME = "example_name";
```

Comment:
Hm... nice idea. Give me some time. I'll try to implement it myself.
I want try to remove service as argument from Request's constructor. And next will try to add Useg-Agent as config param